doc/classes/ProjectSettings.xml

```diff
@@ -749,8 +749,20 @@
 		<member name="physics/2d/default_angular_damp" type="float" setter="" getter="" default="1.0">
 		</member>
 		<member name="physics/2d/default_gravity" type="int" setter="" getter="" default="98">
+			The default gravity strength in 2D.
+			[b]Note:[/b] This property is only read when the project starts. To change the default gravity at runtime, use the following code sample in a script inheriting from CanvasItem (directly or indirectly):
```

Comment:
I'd use `[CanvasItem]` (same for next description and for `[Spatial]` later on).